examples/pytorch/graphsaint/modules.py

```diff
@@ -0,0 +1,76 @@
+import torch.nn as nn
+import torch.nn.functional as F
+import torch as th
+import dgl.function as fn
+
+
+class GCNLayer(nn.Module):
+    def __init__(self, in_dim, out_dim, bias=False, order=1,
+                 activation=None, dropout=0, batch_norm=False):
+        super(GCNLayer, self).__init__()
+        self.linear = nn.Linear(in_dim, out_dim, bias=bias)
+        self.order = order
+        self.activation = activation
+        self.dropout = nn.Dropout(dropout)
+        self.batch_norm = batch_norm
+        if batch_norm:
```

Comment:
The original implementation seems to have two different ways to perform batch normalization [here](https://github.com/GraphSAINT/GraphSAINT/blob/6b3465fffcf764b9646bf571e2b4dc5d6aa74f3c/graphsaint/pytorch_version/layers.py#L23).